pkg/hypershift/hypershift.go

```diff
@@ -186,3 +186,19 @@ func SetHostedControlPlaneConditions(hcp *unstructured.Unstructured, operStatus
 	hcp.Object["status"].(map[string]interface{})["conditions"] = conditions
 	return conditions, nil
 }
+
+// GetCAConfigMapAndKey gets the CAConfigMap and key if set through an environment variable
+// otherwise, default to OpenShift service CA
+func GetCAConfigMapAndKey() (string, string) {
```

Comment:
NIT: There is no need to always read the env variable, we can just read it once and add new fields to `HyperShiftConfig` with the defaulting being done in `NewHyperShiftConfig`. See something similar here: 
https://github.com/openshift/cluster-network-operator/blob/23b758650fd6a91bbea398c3a3b429f8d7114b9e/pkg/hypershift/hypershift.go#L21-L2

Reply:
Updated this.